examples/darwin-framework-tool/templates/tests/tests.js

```diff
@@ -57,6 +57,15 @@ function getTests() {
   // TODO: Test_TC_DIAG_TH_NW_2_3 does not work on Darwin for now.
   tests.disable('Test_TC_DIAG_TH_NW_2_3');
 
+  // TODO: Test_TC_CC_9_1 does not work on Darwin for now.
+  tests.disable('Test_TC_CC_9_1');
```

Comment:
@manjunath-grl can you please file an issue?

Reply:
@krypton36 sure